app/grandchallenge/discussion_forums/models.py

```diff
@@ -122,7 +122,12 @@ def assign_permissions(self):
             self.forum.parent_object.participants_group,
             self,
         )
-        # only challenge admins can delete this topic
+        # only challenge admins can change and delete this topic
```

Comment:
What about changing the topic subject separately from the lock? We may want a specific lock topic permission.